### PR TITLE
fix: waf instance support epsId

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:
waf instance support epsId
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/waf_hw/v1/premium_instances/requests.go [hunk 1]
```diff
@@ -33,9 +36,10 @@ type CreateInstanceOpts struct {
 
 // ListInstanceOpts the parameters in the querying request.
 type ListInstanceOpts struct {
-	Page         int    `q:"page"`
-	PageSize     int    `q:"pagesize"`
-	InstanceName string `q:"instancename"`
+	EnterpriseProjectId string `json:"enterprise_project_id,omitempty"`
```
Comment (chengxiangdong) [inline openstack/waf_hw/v1/premium_instances/requests.go:39]:
```suggestion
	EnterpriseProjectId string `q:"enterprise_project_id"`
```

File: openstack/waf_hw/v1/premium_instances/requests.go [hunk 2]
```diff
@@ -63,6 +67,33 @@ func CreateInstance(c *golangsdk.ServiceClient, opts CreateInstanceOpts) (*Creat
 	return nil, err
 }
 
+func generateEpsIdQuery(epsId string) string {
+	return fmt.Sprintf("?enterprise_project_id=%s", epsId)
+}
```
Comment (chengxiangdong) [inline openstack/waf_hw/v1/premium_instances/requests.go:72]:
```suggestion
func generateEpIdQuery(epid string) string {
	if epid == "" {
		return ""
	}
	return fmt.Sprintf("?enterprise_project_id=%s", epid)
}
```

Comment (chengxiangdong) [reply openstack/waf_hw/v1/premium_instances/requests.go:72]: 在这里加上判断，其他函数调用的时候就可以直接使用了

File: openstack/waf_hw/v1/premium_instances/requests.go (hunk 1 above)
Comment (deer-hang) [reply openstack/waf_hw/v1/premium_instances/requests.go:39]: done

File: openstack/waf_hw/v1/premium_instances/requests.go (hunk 2 above)
Comment (deer-hang) [reply openstack/waf_hw/v1/premium_instances/requests.go]: done

**Pull request merged**